front-end/src/app/profile/profile-routing.module.ts

```diff
@@ -1,7 +1,15 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { ProfileComponent } from './profile.component';
 
-const routes: Routes = [{ path: '**', redirectTo: '' }];
+const routes: Routes = [
+  {
+    path: '',
+    component: ProfileComponent,
```

Comment:
This is a little confusing naming the account info component the same as the module it is in. Let's rename the new component "account" (account.component.*.ts) and move it into its own subdirectory: src/app/profile/account